tests/simulationbase.py

```diff
@@ -7,13 +8,15 @@
 import pytest
 import socket
 import time
+from unittest import TestCase
```

Comment:
Don't. It's not needed. Just use `pytest`'s equivalents.

Reply:
nice, old habits die hard ;)